TDM/Plus_Side/map.json

```diff
@@ -63,7 +63,8 @@
 		}
 	],
 	"itemremove": [
-		"leather helmet", "chainmail chestplate", "leather leggings", "leather boots", "arrow"
+		"iron sword", "bow", "arrow", "cooked beef",
+		"leather helmet", "chainmail chestplate", "leather leggings", "leather boots"
```

Comment:
You could easily even this out better by moving the "leather boots" to the first line instead of the second.